src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -46,6 +51,22 @@ def load_arguments(self, _):
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
         c.argument('arc_agent_version', options_list=['--agent-version'], help='Version of agent to update the helm charts to.')
 
+    with self.argument_context('connectedk8s enable-features') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
+        c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
+        c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
+        c.argument('features', features_types, options_list=['--features'], help='Space-separated list of features you want to enable.')
+        c.argument('azrbac_client_id', options_list=['--client-id'], arg_group='Azure RBAC', help='Client ID for enabling Azure RBAC. Specify when enabling azure-rbac.')
+        c.argument('azrbac_client_secret', options_list=['--client-secret'], arg_group='Azure RBAC', help='Client secret for enabling Azure RBAC. Specify when enabling azure-rbac.')
+        c.argument('azrbac_skip_authz_check', options_list=['--skip-azure-rbac-list'], arg_group='Azure RBAC', help='Comma separated list of names of usernames/email/oid. Azure RBAC will be skipped for these users. Specify when enabling azure-rbac.')
```

Comment:
>Azure RBAC [](start = 98, length = 10)

What does making them part of the group really mean? Like can we still have the skip azure rbac list as optional?